Automattic/wp-calypso
# Open contact page in help center from JITM CTA

#### Proposed Changes

* Allow JITM CTAs to trigger action thunks
* Open help center to contact page (see test video below)

#### Testing Instructions



Simple sites

* Make sure you are sandboxed
* Apply the diff: D94719-code
* JITM should show up in
```
/home/
/stats/
/plans/
/themes/
/plugins/
/comments/
```
* Click on CTA
* Help center should open up to the contact page
* **Make sure the back button in help center goes back to root page**

For atomic sites, setup for that from instructions for WoA in D94719-code.

https://user-images.githubusercontent.com/6586048/206457536-f416f178-206c-4aa1-a84d-f0c33b66f0dd.mp4


https://user-images.githubusercontent.com/6586048/206685803-572984b7-f318-40d3-b6dc-cced682692f0.mp4



#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1157
